index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>First project</title>
+  <link rel="stylesheet" href="./css/style.css" />
+</head>
+
+<body>
+  <header class="header">
+    <div class="burger-menu__open-btn js-burger-open-btn">
+      <svg viewBox="0 0 488 488" id="burger-menu" fill="var(--color-red)">
+        <path
+          d="M64.192,108.444h359.617c29.943,0,54.223-24.278,54.223-54.222C478.031,24.278,453.754,0,423.809,0H64.192            C34.248,0,9.969,24.278,9.969,54.222C9.969,84.166,34.248,108.444,64.192,108.444z" />
+        <path
+          d="M423.809,189.778H64.192c-29.944,0-54.223,24.278-54.223,54.222c0,29.943,24.278,54.223,54.223,54.223h359.617c29.945,0,54.223-24.279,54.223-54.223C478.031,214.056,453.754,189.778,423.809,189.778z" />
+        <path
+          d="M423.809,379.557H64.192c-29.944,0-54.223,24.277-54.223,54.222C9.969,463.722,34.248,488,64.192,488h359.617            c29.945,0,54.223-24.278,54.223-54.222C478.031,403.834,453.754,379.557,423.809,379.557z" />
+      </svg>
+    </div>
+    <div class="burger-menu__container js-burger-menu-block">
+
+      <div class="burger-menu__content">
+        <div>
+          <span class="burger-menu__close-btn js-burger-menu-close-btn">&times;</span>
+        </div>
+        <img class="burger-menu__banner" src="./assets/burger-menu.jpg" alt="burger-menu" />
+        <nav class="burger-menu_nav">
+          <ul class="nav-list">
+            <a class="js-clients-link" href="#">
```

Comment:
Wrapping <a> into <li>  violates html specification. It can cause some issues with seo, or with some browsers. It's better to use vice-versa